docs/getting-started/install-endpoint.asciidoc

```diff
@@ -10,23 +10,26 @@ NOTE: To configure the Endpoint Security integration on the {agent}, you must ha
 [[security-before-you-begin]]
 == Before you begin
 
-Depending on the macOS version you're using, macOS may require you to grant full disk access to different kernels, system extensions, or files. See <<sensor-full-disk-access>> for more information.
+If you're using macOS, some versions may require you to grant full disk access to different kernels, system extensions, or files. See <<sensor-full-disk-access>> for more information.
 
 [discrete]
 [[add-security-integration]]
 == Add the Endpoint Security integration
 
-1. In Kibana, select **Security** -> **Endpoints**. If this is not your first time using Elastic Security, select **Management** -> **Integrations** and search for "Endpoint Security."
+. In Kibana, select **Security** -> **Endpoints**. If this is not your first time using Elastic Security, select **Management** -> **Integrations**, then search for and select **Endpoint Security**.
 +
 [role="screenshot"]
 image::images/install-endpoint/security-integration.png[Search result for "Endpoint Security" on the Integrations page.]
 +
-2. On the Endpoints page of the {security-app} or the Endpoint Security integration page (*Management* -> *Integrations*), select **Add Endpoint Security**. The integration configuration page appears.
-3. Select an agent policy for the {agent}. You can select one of the two default policy options, an existing policy, or select **Create agent policy** to create a new one. For more details on Elastic Agent configuration settings, see {fleet-guide}/elastic-agent-configuration.html[Configuration settings].
-4. Configure the Endpoint Security integration with a name and optional description. When the configuration is complete, click **Save integration** in the lower right corner.
+. Select **Add Endpoint Security** on either the Endpoints page of the {security-app} or the Endpoint Security integration page (*Management* -> *Integrations*). The integration configuration page appears.
 +
 [role="screenshot"]
 image::images/install-endpoint/add-elastic-endpoint-security.png[Add Endpoint Security integration page.]
++
+. Configure the Endpoint Security integration with a name and optional description. 
+. Select an agent policy for the {agent}. You can select a default policy options, an existing policy, or select **Create agent policy** to create a new one. For more details on Elastic Agent configuration settings, see {fleet-guide}/elastic-agent-configuration.html[Configuration settings].
+. When the configuration is complete, click **Save and continue** in the lower-right corner.
+. To complete the integration, install the {agent} on your hosts. 
```

Comment:
should we direct the user / link to where they can learn how to install the {agent} on their hosts?

Reply:
Installing the Elastic Agent is covered in the very next section, but we could still revise this line to make that clearer:

> To complete the integration, continue to the next section to install the {agent} on your hosts.

Reply:
perfection